src/service/data_products/samples.py

```diff
@@ -239,17 +254,28 @@ async def get_samples(
     )
     if status_only:
         return _response(dp_match=dp_match, dp_sel=dp_sel)
-    filters = FilterSet(
+    coll = await appstate.arangostorage.get_collection_active(collection_id)
```

Comment:
There's a subtle bug here - in the case of a load version override, a collection may not even exit yet with that load version, let alone be the active collection. In that case, `coll` should be `None` and if any filter parameters are supplied `get_filters` will throw an error. 

In `genome_attribs` this works correctly, but here, if a load ver override is supplied, coll will be pulled anyway and I'm not quite sure what'll happen. At the very least the data pulled may not be the data expected.

`get_load_version_and_processes` should probably be updated to return coll, then things should work ok, but that method is already pretty complex... what I'd really like to do is rethink the data product code structure in the service but there's no time for that

Reply:
👍 updated `get_load_version_and_processes` to return `coll`.

Reply:
At this point that function should probably return a named tuple or something, but this'll do

Reply:
:+1: